src/NotepadNext/docks/SearchResultsDock.h

```diff
@@ -54,6 +54,8 @@ private slots:
     void itemActivated(QTreeWidgetItem *item, int column);
     void itemExpanded(QTreeWidgetItem *item);
 
+    void on_pb_copyResults_released();
```

Comment:
I'm not a fan of using automatic connections. Qt also suggests not doing this any more. Renaming an object would break the callback without any compile time error.

Specifically using `connect(...)` in the constructor is more verbose but is safer since it would fail at compile time.

Reply:
Yes, sorry, QTCreator suggested it and I took the automatic route instead of thinking ;)